common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4250,6 +4250,9 @@ public static enum ConfVars {
     HIVE_SERVER2_AUTHENTICATION_JWT_JWKS_URL("hive.server2.authentication.jwt.jwks.url", "",
         "URL of the file from where URLBasedJWKSProvider will try to load JWKS if JWT is enabled for the\n" +
         "authentication mode."),
+    HIVE_SERVER2_AUTHENTICATION_JWT_JWKS_SKIP_SSL_CERT("hive.server2.authentication.jwt.jwks.skip.ssl.cert", false,
```

Comment:
Since this property only be enabled in test, how about using `hive.in.test` instead?

Reply:
I feel `hive.in.test` is used in unit tests only and some server behaviors are changed. Won't `hive.in.test` interfere other functionalities when we spin up a cluster?

Reply:
I see, thanks for the explanation